cdap-ui/app/cdap/components/Experiments/CreateView/SplitDataStep/index.js

```diff
@@ -37,94 +38,141 @@ const getSplitLogsUrl = (experimentId, splitInfo) => {
   return `${hostPort}${baseUrl}${queryParams}`;
 };
 
-const renderSplitBtn = (experimentId, splitInfo, onClick) => {
-  let isSplitCreated = Object.keys(splitInfo).length;
-  let splitStatus = (splitInfo || {}).status;
-  let isSplitComplete = ['Complete', 'Failed'].indexOf(splitStatus) !== -1;
-  let isSplitFailed = splitStatus === 'Failed';
-  const splitError = () => {
+const getSplitFailedElem = (experimentId, splitInfo) => {
+  return (
+    <span className="split-error-container">
+      Failed to split: Please check {" "}
+      <a href={getSplitLogsUrl(experimentId, splitInfo)} target="_blank"> Logs </a>{" "}
+      for more information
+    </span>
+  );
+};
+
+class SplitDataStep extends Component {
+  state = {
+    shouldShowSplitFailedAlert: false
+  };
+
+  static propTypes = {
+    splitInfo: PropTypes.object,
+    schema: PropTypes.object,
+    createSplitAndUpdateStatus: PropTypes.func,
```

Comment:
All the action creators can be directly used right? Do we need to pass them as props?